core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js

```diff
@@ -57,7 +58,24 @@ export default function ImportProcess() {
 			importContext.dispatch( { type: 'SET_FILE', payload: null } );
 
 			navigateToMainScreen();
-		};
+		},
+		onReady = () => {
+			setShowUnfilteredFilesDialog( false );
+			setStartImport( true );
+		},
+		eventTracking = ( command, event, source, step, eventType = null ) => {
```

Comment:
Also, if you need the step for each event. Why not include it directly here instead of passing it every time?

Reply:
> Also, if you need the step for each event. Why not include it directly here instead of passing it every time?

Correct should pay more attention next time.

Reply:
> I like it. Why create this function only here?

I have added this function to almost all events, the issue ( that I thought about only in the middle of the implementation ) is that this creates another layer of complexity, it does clean the code but it is harder to maintain.
maybe we should modify the util function to receive data in a more readable way :
e.g: 
 
```
							onSortSelectOpen={ () => appsEventTrackingDispatch(
									'kit-library/change-sort-type',
									{
										event: 'kit sort type dropdown',
										source: 'home page',
										action: 'expand',
									},
								)
							}
```

```
							onSortSelectOpen={ () => appsEventTrackingDispatch( {								        
                                                                                  command: 'kit-library/change-sort-type',
										event: 'kit sort type dropdown',
										source: 'home page',
										action: 'expand',
									} )
							}
```

if we arrange it in one line it is more or less the same as 
`eventTracking = ( command, event, source, step, eventType = null ) => `